mm2src/mm2_main/src/rpc/streaming_activations/fee_estimation.rs

```diff
@@ -0,0 +1,61 @@
+//! RPC activation and deactivation for different fee estimation streamers.
+use super::EnableStreamingResponse;
+
+use coins::eth::fee_estimation::eth_fee_events::EthFeeEventStreamer;
+use coins::{lp_coinfind, MmCoin, MmCoinEnum};
+use common::HttpStatusCode;
+use http::StatusCode;
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::{map_to_mm::MapToMmResult, mm_error::MmResult};
+
+use serde_json::Value as Json;
+
+#[derive(Deserialize)]
+pub struct EnableFeeStreamingRequest {
+    pub client_id: u64,
+    pub coin: String,
+    pub config: Option<Json>,
```

Comment:
I guess we normally use structs or enums implementing Deserialize in nested members in rpc requests, instead of Json

Reply:
covered in [4ca4d69](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/4ca4d693783fc2d28a1fb96b553800deba8a097c)